### ISSUE TITLE
Improve JSON format

### ISSUE THREAD

**Issue by lnjX** [gamingh/ghoAepp-server-side#3]
Currently the JSON format is very hard to parse and shouldn't be the format for an API.
Here is a format I'd like to have. Then we/I could try to implement the conversion to it.

```json
{
  "lastUpdated": "2017-12-24T18:21",
  "date": "2018-01-06",
  "data": [
    {"lesson": 4, "course": "5.1", "subject": "Mu", "room": "F71", "teacher": "WieD"},
    {"lesson": 6, "course": "07.2 De E09", "subject": "De", "room": "B10", "teacher": "Berg"},
    {"lesson": 1, "course": "08.12", "subject": "WAT", "room": "D17", "note": "Gruppe Mert", "type": "Fällt aus"}
  ]
}
```

**Comment by jbruechert** [comment]
If layout of the "upstream" PDF isn't changed again, meaning that the columns stay in the same order, it shouldn't be that difficult to archive.

**Comment by lnjX** [comment]
I don't think we still need this issue
